src/ControllerV2/HomeController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\ControllerV2;
+
+use App\Entity\Beneficiaire;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Security\Http\Attribute\IsGranted;
+
+class HomeController extends AbstractController
+{
+    #[IsGranted('ROLE_USER')]
+    public function renderBeneficiaryNav(?Beneficiaire $beneficiary): Response
+    {
```

Comment:
What do you think about renderVaultNav() ? Because it's also used by profesionnals